.github/workflows/build-images-for-tag-release.yaml

```diff
@@ -138,18 +138,41 @@ jobs:
           make yq
-      - name: Read operator bundle image reference
-        id: parsed-operator-bundle
+      - name: Set environment variables
+        id: set_env_variables
+        run: |
+          bash ./utils/release/load_github_envvar.sh
+      - name: Build catalog variables
+        id: catalog-variables
         run: |
-          image=`make print-bundle-image`
-          echo image=$image >> $GITHUB_OUTPUT
+          bundleImg=${{ env.IMG_REGISTRY_HOST }}/${{ env.IMG_REGISTRY_ORG }}/${{ env.OPERATOR_NAME }}-bundle:${{ env.kuadrantOperatorTag }}
+          echo bundleImg=$bundleImg >> $GITHUB_OUTPUT
+          limitadorBundleImg=${{ env.IMG_REGISTRY_HOST }}/${{ env.IMG_REGISTRY_ORG }}/limitador-operator-bundle:v${{ env.limitadorOperatorTag }}
+          echo limitadorBundleImg=$limitadorBundleImg >> $GITHUB_OUTPUT
+          authorinoBundleImg=${{ env.IMG_REGISTRY_HOST }}/${{ env.IMG_REGISTRY_ORG }}/authorino-operator-bundle:v${{ env.authorinoOperatorVersion }}
+          echo authorinoBundleImg=$authorinoBundleImg >> $GITHUB_OUTPUT
+          dnsOperatorBundleImg=${{ env.IMG_REGISTRY_HOST }}/${{ env.IMG_REGISTRY_ORG }}/dns-operator-bundle:v${{ env.dnsOperatorVersion }}
+          echo dnsOperatorBundleImg=$dnsOperatorBundleImg >> $GITHUB_OUTPUT
       - name: Print tags and references
-        run: echo "Operator bundle image tag = ${{ needs.build-bundle.outputs.image }}, Reference in catalog = ${{ steps.parsed-operator-bundle.outputs.image }}"
+        run: |
+          echo "Operator bundle image tag = ${{ needs.build-bundle.outputs.image }}"
+          echo "Reference in catalog = ${{ steps.catalog-variables.outputs.bundleImg }}"
+          echo "Limitador Operator bundle image tag = ${{ steps.catalog-variables.outputs.limitadorBundleImg }}"
+          echo "Authorino Operator bundle image tag = ${{ steps.catalog-variables.outputs.authorinoBundleImg }}"
+          echo "DNS Operator bundle image tag = ${{ steps.catalog-variables.outputs.dnsOperatorBundleImg }}"
       - name: Verify referenced bundle tag matches the bundle tag currently being built
-        if: ${{ needs.build-bundle.outputs.image != steps.parsed-operator-bundle.outputs.image }}
+        if: ${{ needs.build-bundle.outputs.image != steps.catalog-variables.outputs.bundleImg }}
         run: exit 1
       - name: Generate Catalog Content
-        run: make catalog
+        run: |
+          # Pinning the channel used in the catalog to "stable".
+          # It should not matter for catalogs built for a single release
+          make catalog \
+            BUNDLE_IMG=${{ steps.catalog-variables.outputs.bundleImg }} \
+            LIMITADOR_OPERATOR_BUNDLE_IMG=${{ steps.catalog-variables.outputs.limitadorBundleImg }} \
+            AUTHORINO_OPERATOR_BUNDLE_IMG=${{ steps.catalog-variables.outputs.authorinoBundleImg }} \
+            DNS_OPERATOR_BUNDLE_IMG=${{ steps.catalog-variables.outputs.dnsOperatorBundleImg }} \
+            CHANNEL=stable
```

Comment:
This will set the channel to stable to pre-releases as well, no?

Reply:
yes

Reply:
we might want to change that behaviour, since historically we've been assigning the `alpha` or `preview` to those... maybe we can get the first value from the `channels` env variable exported from the release.yaml file? for the time being? We've never published a catalog with many channels so far. What do you reckon?